src/components/SideBySide/styles.module.scss

```diff
@@ -1,17 +1,18 @@
 .sideBySide {
     display: flex;
+    flex-wrap: wrap;
 
     @media screen and (max-width: 996px) {
         flex-flow: column;
     }
 
     @media screen and (min-width: 997px) {
         > :first-child {
-            padding-right: 5px;
+            padding-right: 10px;
         }
 
         > :last-child {
-            padding-left: 5px;
+            padding-right: 5px;
```

Comment:
Just wondering what happened to the sideBySide whilst making these changes

Reply:
I disliked how long lines inside sidebyside would cause this ugly overflow and really small right/left sides. Now it wraps if there is not enough space to display both elements in one row. Similar to how it wraps on mobile devices.

Before:
![Screenshot 2020-12-27 at 10 19 32](https://user-images.githubusercontent.com/4759511/103167627-03d5bf00-482d-11eb-9adf-e657f5ff8a3b.png)

Now:
![Screenshot 2020-12-27 at 10 19 44](https://user-images.githubusercontent.com/4759511/103167630-0b956380-482d-11eb-99c7-57feee87efd3.png)

Reply:
Oh and <DeprecatedInVersion> looks like this:
![Screenshot 2020-12-27 at 11 34 52](https://user-images.githubusercontent.com/4759511/103168900-94b19800-4837-11eb-8d25-38d995429c2c.png)

Reply:
Ah okay, good change, thanks for the pictures 👍